playout/libretime_playout/liquidsoap/1.4/ls_lib.liq

```diff
@@ -95,112 +95,19 @@ def to_live(old,new) =
 end
 
 
-def output_to(output_type, type, bitrate, host, port, pass, mount_point, url, description, genre, user, s, stream, name, channels) =
-    source = ref s
-    def on_error(msg)
-        gateway("stream '#{stream}' '#{boot_timestamp}' --error='#{msg}'")
-        5.
-    end
-    def on_connect()
-        gateway("stream '#{stream}' '#{boot_timestamp}'")
-    end
-
-    stereo = (channels == "stereo")
-
-    if output_type == "icecast" then
-        user_ref = ref user
-        if user == "" then
-            user_ref := "source"
-        end
-        output_mono = output.icecast(
-                    id = "icecast:stream_#{stream}",
-                    host = host,
-                    port = port,
-                    password = pass,
-                    mount = mount_point,
-                    fallible = true,
-                    url = url,
-                    description = description,
-                    name = name,
-                    genre = genre,
-                    user = !user_ref,
-                    on_error = on_error,
-                    on_connect = on_connect)
-
-        output_stereo = output.icecast(
-                    id = "icecast:stream_#{stream}",
-                    host = host,
-                    port = port,
-                    password = pass,
-                    mount = mount_point,
-                    fallible = true,
-                    url = url,
-                    description = description,
-                    name = name,
-                    genre = genre,
-                    user = !user_ref,
-                    on_error = on_error,
-                    on_connect = on_connect)
-        if type == "mp3" then
-            %include "mp3.liq"
-        end
-        if type == "ogg" then
-            %include "ogg.liq"
-        end
-
-        %ifencoder %opus
-        if type == "opus" then
-            %include "opus.liq"
-        end
-        %endif
-
-        %ifencoder %fdkaac
-        if type == "aac" then
-            %include "fdkaac.liq"
-        end
-        %endif
```

Comment:
I didn't want to keep the %if encoder, as it was ignoring it silently. So for example the aac stream would never show up but no error is printed/raised.